docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Fetch Single Task
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>GET tasks/task/:id/</Endpoint>: Fetching a single Task
+
+```json
```

Comment:
If there isn't anything in the body, we should just keep it as `{}` rather than
```
{

}
```
![image](https://user-images.githubusercontent.com/24628243/159191494-9b3a8556-afc9-44c7-93fb-8a67114cbee0.png)

Reply:
it just takes up more space than needed

Reply:
Fixed: Changes are reflected in recent commits

Reply:
Fixed: Changes are reflected in recent commits